test/unit/lib/redcap_test.rb

```diff
@@ -17,15 +17,15 @@ class Redcap < ActiveSupport::TestCase
     assert_equal 5, forms.values.length
```

Comment:
Agree, but it would be good if 5 were a variable up at the top of the file in case things change.

Not everything has to be relative, but variables make it easier to maintain the tests.